index.js

```diff
@@ -14,11 +33,59 @@ const getFindings = async (ECR, repository, tag) => {
     })
 }
 
+/**
+ * Tally findings by severity.
+ * @param {AWS.ECR.ImageScanFinding[]} ignoredFindings
+ * @returns {IgnoredCounts} counts
+ */
+const countIgnoredFindings = (ignoredFindings) =>
+  ignoredFindings.reduce(
+    (counts, finding) => {
+      const updatedCount = { ...counts }
+      const severity = finding.severity.toLowerCase()
+      updatedCount[severity] = counts[severity] + 1
+      updatedCount.total++
+      return updatedCount
+    },
+    { critical: 0, high: 0, medium: 0, low: 0, informational: 0, undefined: 0, total: 0 }
+  )
+
+/**
+ * Returns display text for a severity level.
+ * @param {string} severity
+ * @param {IgnoredCounts} counts
+ * @returns {string}
+ */
+const getCount = (severity, counts) =>
+  counts[severity] ? `(${counts[severity]} ignored)` : ''
+
+/**
+ * Build an array with CVE IDs to ignore in the counts.
+ * @param {string | string[]} list - Comma-separated list, string with newlines or array of CVE IDs.
+ * @returns {string[]} Array of CVE IDs
+ */
+const parseIgnoreList = (list) => {
+  if (Array.isArray(list)) return list // when GitHub Actions allow arrays to be passed in.
+  if (!list) return []
+
+  const ignoreList =
+    list
+      .trim() // remove trailing newlines if any
+      .replace(/\n/g, ',') // replace newlines with commas, if any
```

Comment:
and spaces? So lists like `CVE-A CVE-B` are supported. I think it could be a common mistake to accidentally generate lists like this using the `>` rather than `|` yaml block style.

Reply:
Ideally, I wanted to use an array but yeah, not supported.

I'll change the regex to `/\n|\s/g` then.
```js
> "CVE-A CVE-B \nCVE-C".replace(/\n|\s/g, ",").split(',').filter(Boolean)
[ "CVE-A", "CVE-B", "CVE-C" ]
```

Reply:
TIL `>` folds newlines to spaces 👍